code/lib/manager-api/src/modules/stories.ts

```diff
@@ -113,6 +119,10 @@ function removeRemovedOptions<T extends Record<string, any> = Record<string, any
   return result;
 }
 
+function toComponentId(storyId: StoryId): ComponentId {
+  return storyId.split('--')[0];
```

Comment:
a little surprised we don't have a utility in `@storybok/csf` for this. no big deal.

Reply:
Agree. I didn't want to mess around with new CSF packages at this moment in time, but can if you disagree.

Reply:
Open a PR for this against the CSF package, but don't merge